src/System.Reflection.MetadataLoadContext/src/System/Reflection/TypeLoading/CustomAttributes/RoPseudoCustomAttributeData.cs

```diff
@@ -19,8 +19,6 @@ internal sealed class RoPseudoCustomAttributeData : RoCustomAttributeData
         //
         internal RoPseudoCustomAttributeData(ConstructorInfo constructor, Func<CustomAttributeArguments> argumentsPromise)
         {
-            CustomAttributeArguments ca = argumentsPromise();
-
```

Comment:
@steveharter, @GrabYourPitchforks, this looks like a bug to me, so I deleted it.  Please let me know if that's incorrect.

Reply:
Since source.dot.net is having trouble resolving across assemblies at the moment I can't tell if this is indirectly a breaking change or not (if the delegate is ever composed over a user provided callback and now it won't fire when it used to).  (And having left that as a reminder of a thing to look for, I'll leave it to the area owners)

Reply:
@steveharter agreed offline that these look like bugs to be fixed.